code/game/machinery/cloning.dm

```diff
@@ -404,6 +404,25 @@
 			icon_state = "pod_0"
 		use_power(200)
 
+REGISTER_BUFFER_HANDLER(/obj/machinery/clonepod)
+
+DEFINE_BUFFER_HANDLER(/obj/machinery/clonepod)
+	if(istype(buffer, /obj/machinery/computer/cloning))
+		if(get_area(buffer) != get_area(src))
+			to_chat(user, "<font color = #666633>-% Cannot link machines across power zones. Buffer cleared %-</font color>")
+			FLUSH_BUFFER(buffer_parent)
+			return
+		to_chat(user, "<font color = #666633>-% Successfully linked [buffer] with [src] %-</font color>")
+		var/obj/machinery/computer/cloning/comp = buffer
+		if(connected)
+			connected.DetachCloner(src)
+		comp.AttachCloner(src)
+	else if (TRY_STORE_IN_BUFFER(buffer_parent, src))
+		to_chat(user, "<font color = #666633>-% Successfully stored [REF(src)] [name] in buffer %-</font color>")
+	else
+		return NONE
```

Comment:
the returning of COMPONENT_NO_AFTERATTACK, null, FALSE, and NONE is relatively inconsistent in these procs

could it be standardized?

Reply:
I'll make any signals that want to return no result 0 and any signal which does not expect return values not have return (null)

Reply:
Oh yea that was super inconsistent